docs/middleware.md

```diff
@@ -0,0 +1,64 @@
+# Middleware
+
+## Express Routing
+
+Underneath it all, Google Cloud Functions written in Node uses the minimalist web framework [Express](https://github.com/expressjs/express) via the [Functions Framework](https://github.com/GoogleCloudPlatform/functions-framework-nodejs).
+
+> In fact, your functions are already using express with the Functions Framework (whether you know it or not), so you aren’t adding an extra dependancy.
+
+This is makes you can writte your Cloud Function in Express style. For example
+
+```js
+const express = require("express");
+
+// Create an Express object and routes (in order)
+const app = express();
+app.use("/users/:id", getUser);
+app.use("/users/", getAllUsers);
+app.use(getDefault);
+
+// Set our GCF handler to our Express app.
+exports.users = app;
+```
+
+## Middleware
+
+You can using Express built middleware directly. For example using cors in our app
```

Comment:
- CORS should be capitalized.
- Use a colon at the end of the sentence, `For example, CORS in our app:`